src/System.Windows.Extensions/src/System/Drawing/Printing/MarginsConverter.cs

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+//
+// System.Drawing.MarginsConverter.cs
+//
+// Author:
+//   Dennis Hayes (dennish@Raytek.com)
+//   Herve Poussineau (hpoussineau@fr.st)
+//
+// (C) 2002 Ximian, Inc
+//
+// Copyright (C) 2004 Novell, Inc (http://www.novell.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
I thought you have said that you are going to use the referencesources implementation instead.

If you are going to use the referencesource implementation, you do not need any of this.